src/Shell/Task/ModelTask.php

```diff
@@ -253,33 +247,46 @@ public function applyAssociations($model, $associations)
     public function findBelongsTo($model, array $associations)
     {
         $schema = $model->schema();
+        $tables = $this->listAll();
+
         foreach ($schema->columns() as $fieldName) {
             if (!preg_match('/^.*_id$/', $fieldName)) {
                 continue;
             }
 
+            $tmpModelName = $this->_modelNameFromKey($fieldName);
```

Comment:
Does this really work if `$fieldName` is `parent_id`? In the original behavior, this statement seems to appear `else` block of `$fieldName === 'parent_id'` check.